unfccc.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UNFCCC</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/unfccc</id>
+    <link rel="self" href="http://www.zotero.org/styles/unfccc"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
```

Comment:
I uploaded the style guide to onedrive and updated the link ("unfccc-with-doc" branch). So now anyone can access it with that link. Pages relevant to the citation style are 33-37. This section will probably undergo revision later this year or next year. I'll update the online file at that point.